content/40.inequality.md

```diff
@@ -1,67 +1,125 @@
+However, it is well documented that these health conditions occur at higher rates in communities of color.<!-- To Do: Add citation-->
+Additionally, social factors such as poverty, housing instability, pollution, and incarceration are likely to play a role in driving discrepancies among racial and ethnic groups and COVID-19 outcomes.<!-- To Do: Add citation-->
+Such social factors have become determinants of individual- and community-level risks related to COVID-19 and are therefore also a critical component of any analysis of COVID-19 pathology.
+
+
+Here, we argue that social, economic, and environmental factors shape an individual's biology in ways that intersect with the pathology of COVID-19.
+In particular, the social factors associated with COVID-19 drive chronic stress, which can in turn alter an individual's biology in ways that are likely to make them more susceptible to COVID-19.
+Because the sources of chronic stress that are most commonly associated with COVID-19 susceptibility are promoted by systemic racism, we argue that it is the experience of racism, not an individual's ancestry, that is associated with more severe outcomes of COVID-19.
+While some efforts have been made to acknowledge ethnicity and race as correlates of social and environmental influences (e.g., [@url:https://www.cdc.gov/coronavirus/2019-ncov/downloads/covid-data/hospitalization-death-by-race-ethnicity.pdf]), rarely is the potential biological basis for these connections explored.
+Therefore, COVID-19 risk must be considered in the context of not only social and environmental but also biological mechanisms through which systemic racism can drive the heavy toll of the pandemic on communities of color.
+
+In this review, we seek to contextualize the social factors associated with COVID-19 severity and to explore how the biological implications of chronic stress intersect with the pathogenesis of SARS-CoV-2.
+First, we outline the potential biological basis of several risk factors that have been identified for COVID-19.
+While the term risk factors implies that they themselves predispose an individual to increased susceptibility of infection and more severe COVID-19 outcomes, we argue that this is a more nuanced question where the interrelatedness of these factors and their association with poverty and racism must be considered.
+Next, we discuss how the conditions presented by systematic discrimination may predispose some individuals both to exposure to SARS-CoV-2 and to more severe outcomes of the disease it causes.
+Many of the social factors associated with elevated allostatic load are also correlated with the likelihood that an individual will be exposed to SARS-CoV-2 in the first place.
```

Comment:
define the first instance of the term "allostatic load"? Could change this sentence to "Many of the social factors associated with the negative physiological consequences of chronic stress (known as "allostatic load") are also correlated with the likelihood that an individual..."